Replaced Button and Branding

AnthonyHoward opened the pull request:
- HTML button replaced with a asp:button to add functionality in the
  code behind
- Woodhaven Senior Community added in Site.Master to aid in branding

AnthonyHoward committed:
Button replaced with a asp:button to add functionality

- HTML button replaced with a asp:button to add functionality in the
code behind
- Woodhaven Senior Community added in Site.Master to aid in branding

AnthonyHoward committed:
Added Page Functionality

- Provides each webpage with a single access point to meals of the week
- A new MealsOfTheWeek class has been created
- Data in the Meals of the week class is the central location for all
meal specials throughout the webpage

svenoaks commented:
The naming conventions for the properties is a bit off in MealsOfTheWeek. Property name should just be MondayMeal with a capital, it's not a method (and method names are capitalized by convention in C# also). 

But really all you really need in this case:

`public string MondayMeal { get; }`

https://msdn.microsoft.com/en-us/library/bb384054.aspx

to avoid a page of get spam. Then you can only set it in the constructor. If you added `private set; 
', you could change it after. Can you change that before merging?

AnthonyHoward committed:
Naming Convention Fixed.  Code simplified.

I took in the advise given and fixed portions of the code. I have also
renamed some elements to more appropriate values. Portions of the code
were simplified. All attached documents need to be updated in the
Master.
